src/app/client/src/app/modules/content-search/components/no-result/no-result.component.html

```diff
@@ -4,7 +4,7 @@
       <div>
         <img src="./assets/images/group.svg" width="90px" height="90px">
       </div>
-      <div class="fmedium board-title my-24">{{title}}</div>
+      <div class="fmedium board-title my-24">{{title}} <span>for {{filters.gradeLevel[0]}}</span></div>
```

Comment:
what happens if the filters object is undefined

Reply:
added if check for filters